blog-posts/git-basic-2/git-basic-2.md

```diff
@@ -49,10 +49,30 @@ git merge my-branch
 Using this you were merging "my-branch" branch into the current branch.
 If the two branches you try to merge both have change the same part of the code, Git won't be able to know which version to keep and he will stop the merge, asking you to resolve the conflict manually. You can also use a tool to do this.
 
+
 ## Git on remote
 
 Git works really fine on local, but it also works on remote. Remote state is another repository, is a version of your project that is hosted on the internet. It is really helpful when we are working on a team, and more persons are working on the same project. There are some repository hosting services like [Github](https://github.com) or [Bitbucked](https://bitbucket.org/) in which you can store a remote repository, so you can manage your remote repository easily.
 
+
+## Saving your projects in remote
+
+To add your project, you have to create the remote repository in one of the platforms mentioned (github, bitbucked...) to get an URL. Once you have this, you need to go to the terminal, make sure you are in your project and use the following command
+
+```
+git remote add origin https://user/repo.git 
+```
+
+'User' is your username in git, and 'repo' is the name of your remote repository. If you want to check which remote your project is you can use
```

Comment:
this url description does not look similar to this one: https://github.com/ThisIsItz/ThisIsDev.git

Also, I do not know if all repos are the same, so probably is better avoid url description and say something like... Take the URL facilitated by your remote repository service